src/main/java/RegistrationService.java

```diff
@@ -0,0 +1,63 @@
+import java.util.Map;
+import java.util.Scanner;
+
+public class RegistrationService {
```

Comment:
Inside of the RegistrationService you are depending on both userManagement and loginService. You could create a new class for example UserService which would then call the methods from the two class, and have everything separated that way